miso-web/src/main/webapp/scripts/list_completions.js

```diff
@@ -0,0 +1,139 @@
+    return [];
+  },
+  createColumns : function(config, projectId) {
+    var nonZero = function(items) {
+      return items.some(function(x) {
+        return x != 0;
+      });
+    }
+    return [
+        {
+          "sTitle" : "Name",
+          "mData" : "pool.id",
+          "mRender" : function(data, type, full) {
+            return "<a href=\"/miso/pool/" + data + "\">" + full.pool.name + "</a>";
+          },
+          "iSortPriority" : 1,
+          "include" : !config.poolId
+        },
+        {
+          "sTitle" : "Alias",
+          "mData" : "pool.alias",
+          "mRender" : function(data, type, full) {
+            return "<a href=\"/miso/pool/" + full.pool.id + "\">" + data + "</a>";
+          },
+          "iSortPriority" : 0,
+          "include" : !config.poolId
+        }, {
+          "sTitle" : "Platform",
+          "mData" : "parameters.platform.instrumentModel",
+          "bSortable" : false,
+          "iSortPriority" : 0,
+          "include" : true
+        }, {
+          "sTitle" : "Sequencing Parameters",
+          "mData" : "parameters.name",
+          "iSortPriority" : 0,
+          "include" : true
+        
+        }, {
+          "sTitle" : "Completed",
+          "mData" : "completed",
+          "bSortable" : false,
+          "iSortPriority" : 0,
+          "include" : true,
+          "visibilityFilter" : nonZero
+        
+        }, {
+          "sTitle" : "Failed",
+          "mData" : "failed",
+          "bSortable" : false,
+          "iSortPriority" : 0,
+          "include" : true,
+          "visibilityFilter" : nonZero
+        }, {
+          "sTitle" : "Requested",
+          "mData" : "requested",
+          "bSortable" : false,
+          "iSortPriority" : 0,
+          "include" : true,
+          "visibilityFilter" : nonZero
+        }, {
+          "sTitle" : "Running",
+          "mData" : "running",
+          "bSortable" : false,
+          "iSortPriority" : 0,
+          "include" : true,
+          "visibilityFilter" : nonZero
+        }, {
+          "sTitle" : "Started",
+          "mData" : "started",
+          "bSortable" : false,
+          "iSortPriority" : 0,
+          "include" : true,
+          "visibilityFilter" : nonZero
+        }, {
+          "sTitle" : "Stopped",
+          "mData" : "stopped",
+          "bSortable" : false,
+          "iSortPriority" : 0,
+          "include" : true,
+          "visibilityFilter" : nonZero
+        }, {
+          "sTitle" : "Unknown",
+          "mData" : "unknown",
+          "bSortable" : false,
+          "iSortPriority" : 0,
+          "include" : true,
+          "visibilityFilter" : nonZero
+        }, {
+          "sTitle" : "Remaining",
+          "mData" : "remaining",
+          "iSortPriority" : 0,
+          "include" : true,
+          "visibilityFilter" : nonZero
+        }, {
+          "sTitle" : "Last Updated",
+          "mData" : "lastUpdated",
```

Comment:
It's just date now. Interestingly, this is the date time format we have everywhere (that's a zone not millis), I'm just not sure it ever gets displayed..